UnitySetup/UnitySetup.psm1

```diff
@@ -1238,17 +1237,17 @@ function Get-UnitySetupInstance {
 
     Write-Verbose "Searching `"$BasePath`" for UnitySetup instances..."
     Get-ChildItem -Path $BasePath -Directory -ErrorAction Ignore | 
-        Where-Object { (Get-UnityEditor $_.FullName).Count -gt 0 } | 
-        ForEach-Object {
-            $path = $_.FullName
-            try {
-                Write-Verbose "Creating UnitySetupInstance for $path"
-                [UnitySetupInstance]::new($path)
-            }
-            catch {
-                Write-Warning "$_"
-            }
+    Where-Object { (Get-UnityEditor $_.FullName).Count -gt 0 } | 
```

Comment:
Nit - lots of formatting changes in here making your intented changes harder to single out.

Reply:
Reverted a bunch of the autoformats that occurred - hopefully it's a little cleaner.